### PR TITLE
DB schema: rename 'bad_words' table to: 'bad_word'

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview
Recreates 'bad_words' table with a singular table name: 'bad_word'

### CODE REVIEW

File: lobby-db/src/main/resources/db/migration/V1.07__rename_bad_words_table.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+create table bad_word as select word, now() as date_created from bad_words;
+alter table bad_word add primary key (word);
+alter table bad_word alter column date_created set not null;
+alter table bad_word alter column date_created set default now();
```
Comment (RoiEXLab) [inline lobby-db/src/main/resources/db/migration/V1.07__rename_bad_words_table.sql]: Why aren't you just doing `ALTER TABLE bad_words RENAME TO bad_word`?
And then `ALTER TABLE bad_word ADD COLUMN date_created timestamptz NOT NULL DEFAULT NOW()`
Also I think we used the `COMMENT ON` feature to document what each column does in the db. Not sure if that's still consistently as available.

**Comment by RoiEXLab** [comment]
(The title suggests something different than the description)

**Comment by DanVanAtta** [comment]
>  You didn't drop the old table. Is there a reason for that?

Yes, notably so that the old lobby application can still work. It's not clear yet how we will transition the next release, for now keeping open options is good. The legacy table really does not hurt, we can drop it soon enough.

Sorry for any confusion, the title is written from the perspective of the code. As far as it is concerned 'bad_words' is a ghost and was renamed.

**Comment by RoiEXLab** [comment]
@DanVanAtta Makes sense in general, but for this specific case I don't think it is necessary. If we want to keep the "old" table, we can always recreate it with no loss of information if it turns out we need it at some point.
So in this soecific case it isn't worth keeping the old table IMO

**Comment by DanVanAtta** [comment]
@RoiEXLab not necessarily a matter of just recreating it, but also the use-case of running two lobby instances in parallel. It's pretty easy to keep that door open for now.

**Pull request merged**